### PR TITLE
Support documentation of simulation resources

### LINKED ISSUE #9: Add properties for documenting a simulation tool to an (application) ontology
Add the following properties for documenting a simulation tool to a new or existing ExecFlow application ontology:
- AiiDAPlugin (`aiida_plugin`)
- Command (`command`)
- InstallCommand (`install_command`)

Consider to also add
- InstallFiles (`install_files`) - just copy a list of files without running any installation software. Alternatively, copy and unpack a zip-file.

### PR DESCRIPTION
**Pull request by jesper-friis**

### CODE REVIEW

File: ontoconv/pipelines.py [hunk 1]
```diff
@@ -7,10 +7,30 @@
...
 
+# Extend the recognised keys used by tripper.convert
+RECOGNISED_KEYS = BASIC_RECOGNISED_KEYS.copy()
+RECOGNISED_KEYS.update(
+    {
+        "aiida_plugin": "http://open-model.eu/ontologies/oip#AiidaPlugin",
+        "command": "http://open-model.eu/ontologies/oip#Command",
+        "install_command": (
+            "http://open-model.eu/ontologies/oip#InstallCommand"
+        ),
```
Comment (francescalb) [inline ontoconv/pipelines.py:25]: In which ontology are these concepts defined?

File: ontoconv/pipelines.py [hunk 2]
```diff
@@ -71,11 +91,13 @@ def populate_triplestore(
     with open(yamlfile, encoding="utf8") as f:
         document = yaml.safe_load(f)
```
Comment (francescalb) [inline ontoconv/pipelines.py]:
```suggestion
        document = yaml.safe_load(f)
```
I think that using 'document' here is misleading. While we do load a document it is not the fact that it is a document that is the point. We are loading documentation, so I suggest using 'documentation' instead.

File: ontoconv/pipelines.py [hunk 3]
```diff
@@ -84,6 +106,43 @@ def populate_triplestore(
...
 
+    # Simulation resources
+    simdoc = document.get("simulation_resources", {})
+    for iri, resource in simdoc.items():
+        iri = ts.expand_iri(iri)
+        save_simulation_resource(ts, iri, resource)
+
+
+def save_simulation_resource(ts: Triplestore, iri: str, resource: dict):
+    """Save documentation of simulation tool to the triplestore.
```
Comment (francescalb) [inline ontoconv/pipelines.py]:
```suggestion
    """Save documentation of simulation tools to the triplestore.
```

File: ontoconv/pipelines.py [hunk 4]
```diff
@@ -84,6 +106,43 @@ def populate_triplestore(
...
+
+def load_simulation_resource(ts: Triplestore, iri: str):
+    """Loads documentation of simulation tool from the triplestore.
+
+    Arguments:
+        ts: Tripper triplestore documenting the simulation tools.
+        iri: IRI of the simulation tool.
+
+    Returns
+        A dict with attriute access documentating the simulation tool.
```
Comment (francescalb) [inline ontoconv/pipelines.py]:
```suggestion
        A dict with attribute access documentating the simulation tool.
```

**Review by francescalb — CHANGES_REQUESTED** [review]
Missing test for save_simulation_resources

File: ontoconv/pipelines.py (hunk 1 above)
Comment (jesper-friis) [reply ontoconv/pipelines.py:25]: No ontology has been created yet. Added an issue: https://github.com/H2020-OpenModel/OntoConv/issues/9

File: ontoconv/pipelines.py (hunk 2 above)
Comment (jesper-friis) [reply ontoconv/pipelines.py]: Agree. Updated

**Comment by jesper-friis** [comment]
Merging to master after discussing the PR with Francesca

**Pull request merged**